Adaptors/SQS/src/PushQueueStorage.cs

```diff
@@ -49,7 +49,6 @@ public PushQueueStorage(AmazonSQSClient           client,
     client_  = client;
     options_ = options;
     logger_  = logger;
-
```

Comment:
this should not be modified in this pr. Otherwise, formatting will complain for indentation